### ISSUE TITLE
Steam is not running. Please start your Steam client and login.

### ISSUE THREAD

**Issue by upl1nk-v01d** [callofduty4x/CoD4x_Server#332]
With upgrading my COD4x client and server to version 20.1 and joining to my dedicated server, I'm encountering this error message:
"Steam is not running. Please start your Steam client and login. Reconnect to this server when logged in."
With version 19.3 there was no such problems.
I launched 19.3 client without steam through linux wine.
With new version I cannot join any servers, even mine.

**Comment by upl1nk-v01d** [comment]
Ok, somehow with struggling and crawling I managed to connect to my linux dedicated server (v20.1) with windows client exe (v20.1) via linux wine v4.21. 
I removed command line parameter "+set fs_game mods/[myMod]" from my client's exe shortcut.
Connecting via any other client's wine versions, server kick me out with steam error message.
On 19.3 client and server version there wasn't such issues.

**Comment by upl1nk-v01d** [comment]
Is there a way to prevent server and client from auto upgrade at launch?
I want to downgrade COD4x back to good old 19.3 version.

**Comment by IceNinjaman** [comment]
Maybe you had not installed a steam_api.so onto your older server?

**Comment by upl1nk-v01d** [comment]
I deleted both steam_api.so files from basepath and homepath.
But by launching server, it automatically downloads deleted files, including steam_api.so file.
How to prevent the server from downloading files?

**Comment by upl1nk-v01d** [comment]
I just tested my server without steam_api.iso by disconnecting my server's pc temporarily from internet to prevent server itself downloading this deleted steam_api.so file.
Everything just worked as expected. I connected to my server with parameter 'fs_game' on client side without any error. :)
Is there any other way to prevent the server downloading missing files, including this steam_api.so file?
